src/py/flwr/driver/app.py

```diff
@@ -113,7 +113,6 @@ def start_driver(  # pylint: disable=too-many-arguments, too-many-locals
         root_certificates = Path(root_certificates).read_bytes()
     driver = GrpcDriver(driver_service_address=address, certificates=root_certificates)
     driver.connect()
```

Comment:
Given that we use `Driver` below, why do we have `GrpcDriver` here?

Reply:
Removed. Thanks for spotting it.